docs/data-sources/infoblox_zone_forward.md

```diff
@@ -0,0 +1,85 @@
+# Zone Forward Data Source
+
+Use the `infoblox_zone_forward` data source to retrieve the following information for Forward Zone if any, which are managed by a NIOS server:
+
+* `fqdn`: The name of this DNS zone. For a reverse zone, this is in “address/cidr” format. Example: `11.10.0.0/24`. For other zones, this is in FQDN format. Example: `demozone.com` This value can be in unicode format.
+* `view`: The name of the DNS view in which the zone resides. Example: `external`.
+* `zone_format`: Determines the format of corresponding zone. Valid values are `FORWARD`, `IPV4` and `IPV6`.
+* `comment`: The Description of Forward Zone Object. Example: `random forward zone`.
+* `ext_attrs`: the set of extensible attributes of the record, if any. The content is formatted as string of JSON map. Example: `"{\"Location\":\"unknown\",\"TestEA\":\"ZoneTesting\"}"`.
+
```

Comment:
Add following fields that can be retreived by datasource -
ns_group, external_ns_group, forward_to, forwarders_only, forwarding_servers, disable.